SoftLayer/CLI/event_log/get.py

```diff
@@ -0,0 +1,47 @@
+"""Get Event Logs."""
+# :license: MIT, see LICENSE for more details.
+
+import json
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+COLUMNS = ['event', 'label', 'date', 'metadata']
```

Comment:
- It is a little annoying when looking the table and there are metadata information on it, maybe it would be better if metadata is an optional column or you could also create a new command which could be used to see the metadata information through the  `traceId` value, similar to `slcli vs detail`

- I suggest you to remove the braces {  } if metadata information will be in the table. 

- I suggest to rename `label` by `object` and recommend to add `object type` and `username` columns to the table, this would help the customers to identify what user made the action/event and in what kind of object, for example I remember that a customer wanted to know what user powered on/off an specific virtual server

Reply:
With commit d21cbd1, these suggestions should be implemented.